eq-author/cypress/utils/index.js

```diff
@@ -90,9 +90,13 @@ export const addQuestionPage = (title = "hello world") => {
   });
 };
 
-export const buildMultipleChoiceAnswer = labelArray => {
+export const buildMultipleChoiceAnswer = (labelArray, label) => {
```

Comment:
It's better to pass parameters as an object as discussed here: https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

Reply:
I had a look into it and and its not a boolean but an optional label for the question. I have made a slight improvement to make it obvious its optional by setting a default. I would appreciate your feedback.

Reply:
I find it more powerful passing objects, rather than parameters as you gain the benefits that javascript objects have such as (de-structuring, spreading, un-ordered keys etc.).  I appreciate this might just be a personal preference so will mark this as resolved.